test/JDBC/input/test_conv_int_to_varbinary_binary-vu-prepare.sql

```diff
@@ -0,0 +1,75 @@
+CREATE VIEW int_to_varbinary_v1
+AS SELECT convert(varbinary(2), 38)
+GO
+
+CREATE VIEW int_to_varbinary_v2
+AS SELECT try_convert(varbinary(3), 38)
+GO
+
+CREATE VIEW int_to_binary_v1
+AS SELECT convert(binary(2), 38)
+GO
+
+CREATE VIEW int_to_binary_v2
+AS SELECT try_convert(binary(3), 38)
+GO
+
+CREATE FUNCTION int_to_varbinary_f1()
+RETURNS sys.varbinary(2)
+AS
+BEGIN
+    RETURN convert(varbinary(2), 38)
+END
+GO
+
+CREATE FUNCTION int_to_varbinary_f2()
+RETURNS sys.varbinary(3)
+AS
+BEGIN
+    RETURN try_convert(varbinary(3), 38)
+END
+GO
+
+CREATE FUNCTION int_to_binary_f1()
+RETURNS sys.varbinary(2)
+AS
+BEGIN
+    RETURN convert(binary(2), 38)
+END
+GO
+
+CREATE FUNCTION int_to_binary_f2()
+RETURNS sys.varbinary(3)
+AS
+BEGIN
+    RETURN try_convert(binary(3), 38)
+END
+GO
+
+CREATE PROCEDURE int_to_varbinary_p1
+AS
+BEGIN
+    SELECT convert(varbinary(2), 38)
+END
+GO
+
+CREATE PROCEDURE int_to_varbinary_p2
+AS
+BEGIN
+    SELECT try_convert(varbinary(3), 38)
+END
+GO
+
+CREATE PROCEDURE int_to_binary_p1
+AS
+BEGIN
+    SELECT convert(binary(2), 38)
+END
+GO
+
+CREATE PROCEDURE int_to_binary_p2
+AS
+BEGIN
+    SELECT try_convert(binary(3), 38)
+END
+GO
```

Comment:
Please add testcases for [var]binary(MAX) as well.

Reply:
On TSQL
```
1> SELECT try_convert(binary(max), 38)
2> go
Msg 156, Level 15, State 1, Server EC2AMAZ-T4MF7PS, Line 1
Incorrect syntax near the keyword 'binary'.
```